topics/proteomics/tutorials/metaquantome-data-creation/tutorial.md

```diff
@@ -0,0 +1,642 @@
+>                - *"Specify Column Names (comma-separated list)"*: `peptide,go_ec`
+>    - *"SQL Query to generate tabular output"*: 
+`SELECT *
+FROM ec`
+>    - *"include query result column headers"*: `Yes`
+>
+> 2. Rename file as go_ec
+{: .hands_on}
+
+
+## *Filter* - Biological Functions
+
+The filter tool allows restriction of the dataset using simple conditional statements. This step is used to filter out the GO terms with biological processes and the corresponding number of peptides associated with these terms.
+
+
+> ### {% icon hands_on %} Hands-on: Extracting biological processes for metaQuantome
+>
+> 1. {% tool [Filter](Filter1) %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `Unipept_Function` (output of **Query Tabular** {% icon tool %})
+>    - *"With following condition"*: `c5=='biological process'`
+>    - *"Number of header lines to skip"*: `1`
+>
+> 2. Rename file as `go_bp`
+{: .hands_on}
+
+
+![Biological Processes](../../images/biologicalprocess.png){: width="60%"}
+
+## *Filter* - Cellular components
+
+This step is used to filter out the GO terms with cellular components and the corresponding number of peptides associated with these terms.
+
+> ### {% icon hands_on %} Hands-on: Extracting cellular component for metaQuantome
+>
+> 1. {% tool [Filter](Filter1) %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `Unipept_Function` (output of **Query Tabular** {% icon tool %})
+>    - *"With following condition"*: `c5=='cellular component'`
+>    - *"Number of header lines to skip"*: `1`
+>
+> 2. Rename file as `go_cc`
+{: .hands_on}
+
+
+![Cellular-Component](../../images/cellularcomponent.png){: width="60%"}
+
+## *Filter* - Molecular Function
+
+This step is used to filter out the GO terms with molecular function and the corresponding number of peptides associated with these terms.
+
+> ### {% icon hands_on %} Hands-on: Extracting molecular function for metaQuantome
+>
+> 1. {% tool [Filter](Filter1) %} with the following parameters:
+>    - {% icon param-file %} *"Filter"*: `Unipept_Function` (output of **Query Tabular** {% icon tool %})
+>    - *"With following condition"*: `c5=='molecular function'`
+>    - *"Number of header lines to skip"*: `1`
+> 2. Rename file as `go_mf`
+{: .hands_on}
+
+
+![Molecular-Function](../../images/molecularfunction.png){: width="60%"}
+
+# **Conclusion**
+{:.no_toc}
+
+This completes the walkthrough of the metaQuantome data creation workflow .This tutorial is a guide to have datasets that are metaQuantome ready/compatible and can be used for metaproteomics research. We have incorporated only two conditions in this workflow but users can use as many as they want. Researchers can use this workflow with their data also, please make sure the tool parameters and the workflow will be needed to be modified accordingly.
+
+Please look at the following tutorials in this Metaproteomics series:
```

Comment:
You can also put recommended follow-up tutorials in the metadata, and they will get rendered nicely in a box at the end, like:

![image](https://user-images.githubusercontent.com/2563865/96109144-a513d100-0ede-11eb-973b-22183b2244df.png)

which was done by adding the following metadata:

```
follow_up_training:
    -
        type: "internal"
        topic_name: transcriptomics
        tutorials:
            - rna-seq-viz-with-heatmap2
            - rna-seq-viz-with-volcanoplot
            - rna-seq-genes-to-pathways

```